internal/account/service.go

```diff
@@ -47,11 +54,37 @@ func (a *AccountService) RegisterAccount(ctx context.Context, spec RegisterAccou
 	return a.accountDBAccessor.RegisterAccount(ctx, account)
 }
 
+func (a *AccountService) Login(ctx context.Context, spec AccountCredentialSpec) (string, error) {
+	var loginError = fmt.Errorf("login failed")
+
+	// Find the account by email
+	account, err := a.accountDBAccessor.FindAccountByEmail(ctx, spec.Email)
+	if err != nil {
+		return "", loginError
```

Comment:
I think it would help debugging in the future if we log the errors outs for each verification step
```suggestion
	if err != nil {
	        log.Printf("account not found: %s, spec.Email)
		return "", ErrLoginFailed
```